tests/iam_policies/user-role.cfn.yaml

```diff
@@ -303,6 +303,7 @@ Resources:
               - ec2:CreateSecurityGroup
               - ec2:CreateSnapshot
               - ec2:CreateTags
+              - ec2:DeleteTags
```

Comment:
Could you please double check if we need to modify:

* https://github.com/aws/aws-parallelcluster/blob/aaa9616e81366919511bfa31ab538468d8497194/cloudformation/policies/parallelcluster-policies.yaml#L364
* https://github.com/aws/aws-parallelcluster/blob/aaa9616e81366919511bfa31ab538468d8497194/tests/integration-tests/tests/iam/test_iam.py#L522
* https://github.com/aws/aws-parallelcluster/blob/aaa9616e81366919511bfa31ab538468d8497194/tests/iam_policies/user-role.cfn.yaml#L305

Reply:
No, you right: we need to update the public policies in [aws-parallelcluster/cloudformation/policies/parallelcluster-policies.yaml](https://github.com/aws/aws-parallelcluster/blob/aaa9616e81366919511bfa31ab538468d8497194/cloudformation/policies/parallelcluster-policies.yaml#L364) because they are consumed by PCAPI and PCUI.

Reply:
Iterated